monkey/monkey_island/cc/services/telemetry/zero_trust_tests/machine_exploited.py

```diff
@@ -0,0 +1,46 @@
+from common.data.zero_trust_consts import *
+from monkey_island.cc.models import Monkey
+from monkey_island.cc.models.zero_trust.event import Event
+from monkey_island.cc.models.zero_trust.finding import Finding
+
+
+def test_machine_exploited(telemetry_json):
```

Comment:
Split to 2 functions:
1. gets telemetry_json and calls the second. This function should be in exploit.py
2. gets current_monkey, target_ip_addr, exploiter, timestamp, result and performs current logic.

Reply:
Split. I prefer to leave the parsing of the json here.